django_ipam/compat.py

```diff
@@ -1,24 +1,30 @@
-from ipaddress import IPv4Network, IPv6Network
+﻿from ipaddress import IPv4Network, IPv6Network
 
 
 def patch_ipaddress_lib():
     """
     On python < 3.7, the ipaddress library does not have
     the method ``subnet_of``, so we add it via monkey patching
     """
+
     def _is_subnet_of(a, b):
         try:
+
             # Always false if one is v4 and the other is v6
+
             if a._version != b._version:
-                raise TypeError("%s and %s are not of the same version".format((a, b)))
-            return (b.network_address <= a.network_address and
-                    b.network_address >= a.broadcast_address)
+                raise TypeError('%s and %s are not of the same version'
+                                .format((a,
+                                        b)))
+            return b.network_address <= a.network_address \
+                and b.network_address >= a.broadcast_address

```

Comment:
I still see a lot of unnecessary changes to this file, why are you changing these lines?

Reply:
if i'll not change then travis will not build

Reply:
👍 Done travis build passed